tcms/testruns/views.py

```diff
@@ -173,7 +173,7 @@ def new(request, template_name='run/new.html'):
                         for value_id in env_values:
                             args.append(EnvRunValueMap(run=test_run, value_id=value_id))
 
-                EnvRunValueMap.objects.bulk_create(args)
+                EnvRunValueMap.objects.bulk_create_with_history(args)
```

Comment:
This is not a simple find-and-replace job:
https://django-simple-history.readthedocs.io/en/latest/reference.html?highlight=bulk_create_with#bulk-creating-a-model-with-history